### PR TITLE
chore: Changing the clean URLs update description in product updates modal

### PR DESCRIPTION
**Pull request by arunvjn**
## Description
Changing the clean URLs update description in product updates modal.

## Type of change
- Description change

## How Has This Been Tested?
- Manual

## Design
https://www.figma.com/file/b3tHFXEoghB4xCbo6FIRnH/Manual-Updates-Clean-URLs

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: chore/updates-modal--text-change 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.74 **(0.02)** | 37.18 **(0.02)** | 35.48 **(0.03)** | 56 **(0.02)**
 :green_circle: | app/client/src/ce/constants/messages.ts | 77.1 **(0.03)** | 100 **(0)** | 32.3 **(0.29)** | 81.92 **(0.05)**
 :green_circle: | app/client/src/pages/Editor/BottomBar/ManualUpgrades.tsx | 91.67 **(2.31)** | 85 **(3.75)** | 76.92 **(4.19)** | 90.2 **(2.7)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>

### CODE REVIEW

File: app/client/src/pages/Editor/BottomBar/ManualUpgrades.tsx [hunk 1]
```diff
@@ -168,6 +201,33 @@ function ManualUpgrades() {
...
+  const { applicationSlug, pageSlug } = useSelector(selectURLSlugs);
+  const location = useLocation();
+
+  const updates = React.useMemo(
+    () => [
+      {
+        name: createMessage(CLEAN_URL_UPDATE.name),
+        shortDesc: createMessage(CLEAN_URL_UPDATE.shortDesc),
+        description: [
+          "All URLs in your applications will be updated to match our new style. This will make your apps easier to find, and URLs easier to remember.",
```
Comment (mohanarpit) [inline app/client/src/pages/Editor/BottomBar/ManualUpgrades.tsx:215]: Is it possible to move this to messages.ts as well? This way all our content copy is in a single place.

**Review by mohanarpit — APPROVED** [review]

**Comment by arunvjn** [comment]
Screenshot 2022-04-04 at 09 25 24
Product updates modal with the new text

**Review by albinAppsmith — APPROVED** [review]

**Pull request merged**